package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jupyter",
     "displayName": "Jupyter",
-    "version": "2021.4.0",
+    "version": "2021.5.0",
```

Comment:
@DavidKutu Yup. Currently our next release (what I just snapped into the release branch) is 2021.4.0 so early April release. Since that is the release branch main is now targeting the release after that which is an early May release.